src/Header.js

```diff
@@ -32,7 +34,12 @@ class Header extends Component {
         }
         if (pageLocation === "settings") {
 			iconElementLeft = <IconButton onTouchTap={Header.goBackInHistory}><IconArrowBack/></IconButton>;
+            iconElementRight = <IconButton href="/create/eventType"><IconAdd/></IconButton>;
         }
+        if (pageLocation === "createEventType") {
+            iconElementLeft = <IconButton onTouchTap={Header.goBackInHistory}><IconArrowBack/></IconButton>;
+            iconElementRight = <div/>;
+		}
 
         let className = "";
```

Comment:
Maybe we could use a more descriptive name here. For example `headerStyle` or `appBarStyle/Styling`.
`className` isn't quite clear to me.
This would also avoid `className={className}` in line 62.